docs/api/sql/Raster-visualizer.md

```diff
@@ -63,3 +63,26 @@ Output:
 | 3   4   5   6|
 ```
 
+
+## Visualize a raster as PNG
+
+Introduction: Returns a PNG byte array that can be stored as PNG using the [sedona function](../Raster-writer/#write-a-binary-dataframe-to-raster-files). This function can only accept pixel data type of unsigned integer.
+
+!!!Note
```

Comment:
This doc should be put in the raster writer section, together with AsGeoTiff. Please also explain what will happen if the user gives double type raster.